autopush/logging.py

```diff
@@ -108,6 +105,23 @@ def __call__(self, event):
             self._output.write(unicode(text))
             self._output.flush()
 
+    def raven_log(self, event):
+        f = event["log_failure"]
+        stack = extra = None
+        tb = f.getTracebackObject()
+        if not tb:
+            # include the current stack for at least some context
+            stack = list(iter_stack_frames())[4:]  # approx.
```

Comment:
Will there ever be a chance that the iter_stack_frames could ever be less than 4? (I know, with the frameworks, fairly remote if possible, but still, wouldn't want to lose a report to an index error.)